homework-submission/homework-week1/App.js

```diff
@@ -0,0 +1,84 @@
+class ListItem extends React.Component {
+  // ListItem component is creating a list item in page
+  render() {
+    const { description, deadline } = this.props;
```

Comment:
You could also get `id` and `done` from `this.props` here. Gives cleaner code in the jsx of this component.

Reply:
Yeah I missed. Now I've added it => https://github.com/HackYourFuture/React/pull/452/commits/19cf1398d231c2d7e32abb8afc265eef15db1b02